static/js/components/UserChip_test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { assert } from 'chai';
+
+import { USER_PROFILE_RESPONSE } from '../constants';
+import R from 'ramda';
+import UserChip from './UserChip';
+import { getPreferredName } from '../util/util';
+import ProfileImage from './ProfileImage';
+
+describe('UserChip', () => {
+  let clone;
+
+  beforeEach(() => {
+    clone = R.clone(USER_PROFILE_RESPONSE);
```

Comment:
i think i've seen this pattern elsewhere in the codebase, but why is this called `clone`? why not `profile`?